src/gui/mod.rs

```diff
@@ -74,6 +79,8 @@ pub struct Ajour {
     ignored_addons: Vec<(Addon, button::State)>,
     is_showing_settings: bool,
     shared_client: Arc<HttpClient>,
+    needs_update: Option<String>,
```

Comment:
If this was not a typed language it would assume it was a `bool`, alone because of the wording.

Reply:
True... maybe `newer_version`?